package R3;

import U4.c;
import java.io.IOException;

final class X2 implements U4.d {

    static final X2 f5416a = new X2();

    private static final U4.c f5417b;

    private static final U4.c f5418c;

    private static final U4.c f5419d;

    private static final U4.c f5420e;

    private static final U4.c f5421f;

    static {
        c.b bVarA = U4.c.a("detectorOptions");
        C1171t0 c1171t0 = new C1171t0();
        c1171t0.a(1);
        f5417b = bVarA.b(c1171t0.b()).a();
        c.b bVarA2 = U4.c.a("errorCode");
        C1171t0 c1171t02 = new C1171t0();
        c1171t02.a(2);
        f5418c = bVarA2.b(c1171t02.b()).a();
        c.b bVarA3 = U4.c.a("totalInitializationMs");
        C1171t0 c1171t03 = new C1171t0();
        c1171t03.a(3);
        f5419d = bVarA3.b(c1171t03.b()).a();
        c.b bVarA4 = U4.c.a("loggingInitializationMs");
        C1171t0 c1171t04 = new C1171t0();
        c1171t04.a(4);
        f5420e = bVarA4.b(c1171t04.b()).a();
        c.b bVarA5 = U4.c.a("otherErrors");
        C1171t0 c1171t05 = new C1171t0();
        c1171t05.a(5);
        f5421f = bVarA5.b(c1171t05.b()).a();
    }

    private X2() {
    }

    @Override
    public final void a(Object obj, Object obj2) throws IOException {
        throw null;
    }
}
